src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -596,6 +585,16 @@ void closeAllWithTimeout(final Collection<? extends AutoCloseable> closeables) {
 		}
 	}
 
+	private String eraseSlashFromString(String str) {
```

Comment:
***static*** String extractStreamName(***final*** String itemPath) {

Reply:
Done. But do we need to use this method in some other places? Why couldn't it be private?